car/go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/ipld/go-car/car
 
-go 1.13
+go 1.15
```

Comment:
I am curious why not upgrade to `1.16` while we are at it

Reply:
You can think of this version as the absolute minimum that is required to build the module. We generally support the two latest major Go versions, being 1.15.x and 1.16.x right now, so 1.15 is the highest we can set.

Reply:
I see; that makes sense. Thanks